icons/nixos/nixos-original-wordmark.svg

```diff
@@ -0,0 +1,13 @@
+<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I would recommend optimizing **all** the SVGs using the [SVG Viewer](https://www.svgviewer.dev/) (`viewBox` should always be included).

Reply:
Done!